src/bitcoin-cli.cpp

```diff
@@ -1028,6 +1073,64 @@ static void ParseGetInfoResult(UniValue& result)
         result_string += "\n";
     }
 
+    const double mempool_bytes{result["mempool_bytes"].get_real()};
```

Comment:
A function for whole thing? Yes.

Reply:
Yup L1076-L1130 looks like it could be cleanly refactored into another function 👍

Reply:
Sorry Its been more than a month. I was waiting for some progress in PR 21422 before making any changes here as it depends on it. Anyways looked at the code today and this looks alright because everything is inside `ParseGetInfoResult()`